portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithEditMode.testcase

```diff
@@ -225,11 +225,19 @@ definition {
 			MouseOver.mouseOverNotVisible(locator1 = "//div[contains(@class,'portlet-options')]/a[contains(@id,'AssetPublisher')]");
 		}
 
+		task ("Select Page Design") {
+			Click(locator1 = "PageEditor#EDIT_MODE_SELECTOR");
+
+			MenuItem.click(menuItem = "Page Design");
+
+			AssertTextEquals(
+				locator1 = "PageEditor#EDIT_MODE_SELECTOR",
+				value1 = "Page Design");
+		}
+
```

Comment:
Have you confirm with developer this behavior Page Content panel is unavailable in Content Editing mode?